freebsd/ProcessField.h

```diff
@@ -11,6 +11,7 @@ in the source distribution for its full text.
 #define PLATFORM_PROCESS_FIELDS  \
    JID   = 100,                  \
    JAIL  = 101,                  \
+   EMULATION = 102,              \
```

Comment:
Process.h and linux/ProcessField.h don't bother aligning anything, it's all kept flush, so I didn't go and re-align the JID and JAIL. They should probably be de-aligned themselves though for consistency...

Reply:
If you do, have those stylistic changes as a separate commit.

Reply:
I'll open a separate trivial PR for that once this is merged